config.toml

```diff
@@ -44,9 +44,11 @@ max_open_conns = 3
 max_idle_conns = 1
 conn_max_lifetime = 3600
 hosted_databases = ["scratch"]
+hosted_clusters = ["scratch"]
 
 [[users]]
 api_key = "local"
 db_user = "scratch"
 db_name = "scratch"
 db_password = "scratch"
+db_cluster = "scratch"
```

Comment:
I finally came up with a simple solution to how to manage macros. Add 3 config settings here:

- shard_macro_name
- replica_macro_name
- (etc)

Then use these values on [this line](https://github.com/scratchdata/ScratchDB/pull/44/files#diff-93784a35b2de36099e2c63a4b0151733876c40f6ff919d2069ee91843979b846R146). So instead of 

```
ReplicatedMergeTree('/clickhouse/{cluster}/tables/{shard}/{database}/{table}', '{replica}')
```

you could say something like

```
ReplicatedMergeTree('/clickhouse/{cluster}/tables/{shard}/{database}/{table}', '{config.GetReplicaMacroName()}')
```

This would allow us to change the name of the macro per api key. A single server could have multiple independent macro values depending on the clusters that are running.

Reply:
I updated this.

https://github.com/scratchdata/ScratchDB/blob/c820a915179637534f123bf949d38299832acabf/importer/importer.go#L141-L164

Reply:
@poundifdef I updated this.

https://github.com/scratchdata/ScratchDB/blob/c820a915179637534f123bf949d38299832acabf/importer/importer.go#L141-L164